packages/zowe-explorer/src/Profiles.ts

```diff
@@ -894,7 +894,7 @@ export class Profiles extends ProfilesCache {
                     const doNotSaveButton = localize("promptCredentials.doNotSave.button", "Do Not Save");
                     const infoMsg = localize(
                         "promptCredentials.saveCredentials.infoMessage",
-                        "Save entered credentials for future use with profile: {0}? Saving credentials will update the local yaml file.",
+                        "Save entered credentials for future use with profile: {0}?\nSaving credentials will update the local yaml file.",
```

Comment:
The `\n` here doesn't seem to create a newline for me in the pop-up message. I do see that this is in `master` branch, so I think it's okay to go into `next`, but I was curious if others can see its effect.